### PR TITLE
RF | _OrganizationFolderTest> createOrganizationFolderSameItemNameTest 

### PR DESCRIPTION
**Pull request by MariaMoroz**
https://trello.com/c/BHnLADPV/1103-rf-organizationfoldertest-createorganizationfoldersameitemnametest

### CODE REVIEW

**Review by ValeriiSarukhanov — APPROVED** [review]

File: src/test/java/_OrganizationFolderTest.java [hunk 1]
```diff
@@ -28,6 +27,7 @@ public void testCreateOrganizationFolder() {
                 .saveConfigAndGoToProject()
                 .getProjectName();
 
+
```
Comment (ke-de-ve) [inline src/test/java/_OrganizationFolderTest.java]: extra line

File: src/test/java/_OrganizationFolderTest.java [hunk 2]
```diff
@@ -43,9 +43,8 @@ public void testRenameOrganizationFolder() {
         Assert.assertEquals(projectName, VALID_FOLDER_RENAME);
     }
 
-    @Ignore
     @Test(dependsOnMethods = "testRenameOrganizationFolder")
-    public void testCreateOrganizationFolderSameItemName() {
+    public void testCreateOrganizationFolderSameItemName() throws InterruptedException {
```
Comment (ke-de-ve) [inline src/test/java/_OrganizationFolderTest.java:46]: leftover from deleted sleep?  remove throws

**Review by ke-de-ve — CHANGES_REQUESTED** [review]
a couple of changes needed here

File: src/test/java/_OrganizationFolderTest.java (hunk 2 above)
Comment (MariaMoroz) [reply src/test/java/_OrganizationFolderTest.java:46]: done

File: src/test/java/_OrganizationFolderTest.java (hunk 1 above)
Comment (MariaMoroz) [reply src/test/java/_OrganizationFolderTest.java]: done

**Review by ke-de-ve — APPROVED** [review]

**Pull request merged**